addons/scratch-notifier/comment-emojis.js

```diff
@@ -1,7 +1,7 @@
 export default Object.assign(Object.create(null), {
   "meow.png": "[meow emoji]",
   "gobo.png": "[gobo emoji]",
-  "waffle.png": "[waffle emoji]",
+  "waffle.png": "🧇",
```

Comment:
I didn't add it because it was pretty new (Unicode 12.0)

Reply:
I don't think Sora supports emojis at all.